@kiva/kv-components/vue/KvLoanProgressGroup.vue

```diff
@@ -1,11 +1,19 @@
 <template>
 	<figure>
-		<h4
-			class="tw-lowercase tw-mb-0.5"
-			:class="{ 'progress-group-amount-low': amountLow }"
-		>
-			{{ fundingText }}
-		</h4>
+		<div :class="{'tw-flex tw-justify-between': amountGoal}">
```

Comment:
It looks like the `v-if="amountGoal"` should be on this parent `div` instead of the child `h4`.